deb/blockchain-http.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Blockchain HTTP application
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/var/helium/blockchain_http/bin/blockchain_etl foreground
```

Comment:
This is the wrong app.. 

It's also making assumptions about where the service is located. Maybe we should document this?

Reply:
Maybe put this file in config to avoid creating yet another folder.. And add `.sample` to make it very clear it won't work out of the box

Reply:
I was following your `blockchain-etl` patterns

Reply:
makes sense